### DISCUSSION TITLE
Reuploading Content from the Keeli Company Aux Mod

### DISCUSSION THREAD

**Discussion by DartRuffian** [DartRuffian/KeeliCompanyAux#223] [General]
# Permissions
The following individuals and/or units have explicit permission to re-upload *specific* content from the Keeli Company Aux mod. If a steam workshop item is found to be using re-uploaded assets and is not from one of the authors below, let us know as soon as possible so the offending content can be taken down.

- [21st Nova Corps (Discord)](https://discord.gg/21stnovacorps)
  - Loadout item replacer
  - [Intercom](https://github.com/Outer-Rim-Armory/Aux-Mod/tree/main/addons/intercom)

Other entities not on this list do not have permission to re-upload content.
If you would like to request to re-upload some content, do so on this discussion post.

**Comment by ItzMerek56** [comment]
Hey tyrant sent me here so i can get more info on the jetpacks
